core/src/main/scala/org/apache/spark/internal/config/Network.scala

```diff
@@ -49,7 +49,13 @@ private[spark] object Network {
     ConfigBuilder("spark.network.timeoutInterval")
       .version("1.3.2")
       .timeConf(TimeUnit.MILLISECONDS)
-      .createWithDefaultString(STORAGE_BLOCKMANAGER_TIMEOUTINTERVAL.defaultValueString)
+      .createWithDefaultString("15s")
+
+  private[spark] val NETWORK_EXECUTOR_TIMEOUT =
+    ConfigBuilder("spark.network.executorTimeout")
+      .version("1.3.0")
+      .timeConf(TimeUnit.MILLISECONDS)
+      .createWithDefaultString("60s")
```

Comment:
fallback to `NETWORK_TIMEOUT` to preserve existing behavior.

Reply:
Updated. I change `NETWORK_EXECUTOR_TIMEOUT` to `Option`. In addition, I write two tests to test this change.

```scala
private val executorTimeoutMs = sc.conf.get(
  config.STORAGE_BLOCKMANAGER_HEARTBEAT_TIMEOUT
).getOrElse(
  sc.conf.get(Network.NETWORK_EXECUTOR_TIMEOUT) match {
    case Some(executorTimeout) => Utils.timeStringAsMs(s"${executorTimeout}s")
    case None => Utils.timeStringAsMs(s"${sc.conf.get(Network.NETWORK_TIMEOUT)}s")
  }
)
```

* Test
```bash
build/sbt "core/testOnly *HeartbeatReceiverSuite -- -z NETWORK_EXECUTOR_TIMEOUT"
```